src/components/UpdateArticle.jsx

```diff
@@ -11,48 +12,85 @@ const UpdateArticle = (props) => {
-                  name="premium"
-                  value={true}
-                  type="radio"
-                  defaultChecked
-                />
-                <label style={{ display: "inline" }}> Free </label>
-                <input
-                  id="radio-premium"
-                  label="Premium"
-                  name="premium"
-                  value={false}
-                  type="radio"
-                />
-                <label style={{ display: "inline" }}> Premium</label>
-              </Form.Field>
-              <Button id="publish-btn" type="submit">
-                Publish Article
-              </Button>
-              <p id="message">{props.message}</p>
-            </Form>
+          <Grid.Column width={4}>
+            <Container id="form-container">
+            <Link to="/review" id="back-btn">
+              <Button>Back to list</Button>
+            </Link>
+            {props.message === "Article successfully published!" ? (
+              <p id="success-message">{props.message}</p>
+            ) : (
+              <Form
+                className="publishing-form"
+                onSubmit={(e) => props.onSubmitHandler(e)}
+              >
+                <Form.Field>
+                  <label>Category</label>
+                  <Dropdown
+                    selection
+                    id="category"
+                    name="category"
+                    placeholder={selectedArticle.category}
+                    options={categories}
+                  ></Dropdown>
+                </Form.Field>
+                <Form.Field>
+                  <input
+                    id="radio-free"
+                    label="Free"
+                    name="premium"
+                    value={true}
+                    type="radio"
+                    defaultChecked
+                  />
+                  <label style={{ display: "inline" }}> Free </label>
+                  <input
+                    id="radio-premium"
+                    label="Premium"
+                    name="premium"
+                    value={false}
+                    type="radio"
+                  />
+                  <label style={{ display: "inline" }}> Premium</label>
+                </Form.Field>
+                <Form.Field>
+                  <input
+                    id="checkbox-sweden"
+                    name="local"
+                    value={false}
+                    type="checkbox"
+                  />
+                  <label style={{ display: "inline" }}>
+                    {" "}
+                    Local News Sweden{" "}
+                  </label>
+                  <input
+                    id="checkbox-International"
+                    name="international"
+                    value={false}
+                    type="checkbox"
+                  />
+                  <label style={{ display: "inline" }}>
+                    {" "}
+                    International News{" "}
```

Comment:
Is this to get proper spacing, or something else?
If yes, better would be to do margin left and right on them, second best would be to put everything in a block and add spaces to either side {" News "}.
According to screenshot, these labels are kind of on top of eachother, could we do a line break or something to split them up?

Reply:
Hm, I didn't do that spacing, I think we can do without it, with CSS instead. That they are on top of each other is because of the narrow Cypress window, it looks better on fullscreen